lib/content-services/upload/components/file-uploading-list-row.component.html

```diff
@@ -12,7 +12,7 @@
     </span>
 
     <span *ngIf="isUploadVersion()" class="adf-file-uploading-row__version">
-        <mat-chip aria-label="file version" color="primary" disabled>{{
+        <mat-chip color="primary" [title]="'version' + versionNumber" disabled>{{
```

Comment:
is it still accessible without aria-label?

Reply:
guess you are right, aria-label should be used in conjunction with the title attribute